.circleci/src/commands/skip_job.yml

```diff
@@ -6,15 +6,17 @@ parameters:
     default: 'all'
 steps:
   - run:
-      name: Skip the current build if neccassary
+      name: Skip the current build if necessary
       # language=bash
       command: |
         function skip_job {
           build_config_name=$1
           variable_name=SKIP_JOB_${CIRCLE_JOB%-*} # Remove everything after "-". e.g. "check-1" -> check
           if [[ -n ${!variable_name} && ($build_config_name =~ ${!variable_name} || "all" =~  ${!variable_name}) ]]; then
-            echo "This job is skipped because the variable $variable_name is set"
+            message="Job is skipped because the variable $variable_name is set"
+            echo $message
             circleci step halt
+            source tools/notify-mattermost main integreat-app-notifications "$message"
```

Comment:
Why are we using the tool here directly instead of the `notify` command?

Reply:
Because we cannot call the notify command conditionally depending on a environment variable. CircleCI does not support calling commands conditionally on environment variables, as they are evaluated on run time, whereas the conditional-statements are evaulated on compile time.